src/main/java/uk/gov/hmcts/cmc/claimstore/events/claimantresponse/ClaimantResponseActionsHandler.java

```diff
@@ -71,6 +75,14 @@ private boolean isFreeMediationConfirmed(Claim claim) {
             && isOptedForMediation(response);
     }
 
+    private boolean hasClaimantSettledForFullDefense(Claim claim){
+        ClaimantResponse claimantResponse = claim.getClaimantResponse().orElseThrow(IllegalStateException::new);
+        Response response = claim.getResponse().orElseThrow(IllegalArgumentException::new);
+        return claimantResponse.getType() == ClaimantResponseType.ACCEPTATION
+            && response.getResponseType() == ResponseType.FULL_DEFENCE
+            && (isResponseFullDefenceStatesPaid(response) || isFullDefenceDispute(response));
+    }
```

Comment:
The story was just for `DISPUTE` defences, not states paid - this might be changing the behaviour for other intent to proceed scenarios (lines 47 and 49 as I write)

Reply:
agree with @andrewwa-kainos that we should maybe remove : `isResponseFullDefenceStatesPaid(response)` in the OR Clause